data/family/Debian.yaml

```diff
@@ -1,4 +1,5 @@
 ---
+rabbitmq::python_package: 'python3'
```

Comment:
note: these are not alpha sorted; I tried to follow the convention / lack of convention from other files, but happy to reorder some or all of these if necessary.